lib/privatemessages/onMessage.js

```diff
@@ -34,26 +34,28 @@ exports.func = function (args) {
   const onMessage = new events.EventEmitter()
   let waitingForRequest = false
   let latest
-  getMessages({ jar, messageTab: 'Inbox', pageNumber: 0 })
+  getMessages({ jar: jar, messageTab: 'Inbox', pageNumber: 0, pageSize: 1 })
     .then(function (initial) {
       latest = initial.collection[0] ? initial.collection[0].id : 0
-      const notifications = onNotification({ jar })
+      const notifications = onNotification({ jar: jar })
       notifications.on('data', function (name, message) {
-        if (name === 'NotificationStream' && message.Type === 'NewNotification') {
+        if (name === 'MessageNotification' && message.Type === 'Created') {
           if (waitingForRequest) {
             waitingForRequest = false
           } else {
             getMessages({
-              jar,
+              jar: jar,
               messageTab: 'Inbox',
-              pageNumber: 0
+              pageNumber: 0,
+              pageSize: 1
             })
               .then(function (inbox) {
                 const messages = inbox.collection
                 for (let i = messages.length - 1; i >= 0; i--) {
                   const message = messages[i]
                   const id = message.id
                   if (id > latest) {
+                    console.log(id)
```

Comment:
This should probably be removed

Reply:
My bad I didn’t see that when I pushed it. I will go ahead and remove it.